src/routes.ts

```diff
@@ -1,7 +1,9 @@
 import { Chat } from './pages/Chat'
+import { DownloadApp } from './pages/DownloadApp'
 import { Offering } from './pages/Offer'
 
 export const reactPageRoutes = [
   { path: '/hedvig', Component: Chat, exact: true },
   { path: '/offer', Component: Offering, exact: true },
+  { path: '/downloadapp', Component: DownloadApp, exact: true },
```

Comment:
imo `/download` is sufficient :) but maybe no?

Reply:
`/download` = :+1: